pkg/platform/openstack/kuryr_bootstrap.go

```diff
@@ -410,13 +411,19 @@ func ensureOpenStackSg(client *gophercloud.ServiceClient, name, tag string) (str
 
 // Tries to create an OpenStack security group rule on sgId SG. Ignores an
 // error if such rule already exists.
-func ensureOpenStackSgRule(client *gophercloud.ServiceClient, sgId, remotePrefix string) error {
+func ensureOpenStackSgRule(client *gophercloud.ServiceClient, sgId, remotePrefix string, portMin, portMax int) error {
 	opts := rules.CreateOpts{
 		SecGroupID:     sgId,
 		EtherType:      rules.EtherType4,
 		Direction:      rules.DirIngress,
 		RemoteIPPrefix: remotePrefix,
 	}
+	// Let's just assume that we're getting passed -1 when we aren't supposed to set those
+	if portMin >= 0 && portMax >= 0 {
```

Comment:
is it possible for one of them to be -1?

Reply:
-1 is just a mnemonic for "not set it", if my golang understanding isn't completely wrong I would need to create 2 versions of this method otherwise.
I can also add a boolean parameter marking if portMin and portMax should be ignored, if you think that's cleaner?

Reply:
Oh nevermind, I see those are only set from the code. This is fine.